inspire/utils/latex.py

```diff
@@ -0,0 +1,428 @@
+#
```

Comment:
Add in the docstring on top 

```
# -*- coding: utf-8 -*-
```

Like in:
https://github.com/inspirehep/inspire-next/blob/master/inspire/modules/authors/forms.py#L1

And only put one `#` in front of every line.